### DISCUSSION TITLE
Sidekiq dashboard graphs are gone

### DISCUSSION THREAD

**Discussion by pbstriker38** [sidekiq/sidekiq#6095] [Q&A] [answered]
I don't see any console or network errors. Any idea what's going on?

![image](https://github.com/sidekiq/sidekiq/assets/5264240/67cbcc46-1752-4297-a385-df430611661f)

**Comment by mperham** [comment]
I would look at the browser console for JS errors or Content Security violations.

**Reply by pbstriker38** [reply]
There were no console errors or CSP violations. Oddly, the graphs are now working today 🤷‍♂️

**Reply by mperham** [reply, marked as answer]
The Web UI, by default, caches all resources for 24 hours. You might have upgraded Sidekiq versions but had an old version of a web asset in your browser cache which did not work with the new version. Glad it cleared up.

**Reply by pbstriker38** [reply]
Well, that would be it since it broke right after updating and then it now works a day later.
